### PR TITLE
Test EWMH client messages

### PR DESCRIPTION
**Pull request by t-wissmann**
This adds tests for the EWMH client message handling for closing windows
and for starting/stopping mouse dragging operations.

### CODE REVIEW

File: tests/test_ewmh.py [hunk 1]
```diff
@@ -445,3 +446,84 @@ def test_net_supported(hlwm, x11):
...
+    (NetWmMoveResize.SIZE_BOTTOM, False),
+    (NetWmMoveResize.SIZE_BOTTOMLEFT, False),
+])
+def test_initiate_drag(hlwm, mouse, x11, action_nr, is_moving):
+    hlwm.attr.tags.focus.floating = True
+    winHandle, winid = x11.create_client()
+    geo_before = hlwm.attr.clients[winid].floating_geometry()
+    mouse_pos = geo_before.center() + Point(2, 2)
+    mouse.move_to(mouse_pos.x, mouse_pos.y)
+    # x11.display.warp_pointer(mouse_pos.x, mouse_pos.y)
```
Comment (blarz) [inline tests/test_ewmh.py:497]: Leftover debug statement?

**Review by blarz — CHANGES_REQUESTED** [review]
Just one thing

File: tests/test_ewmh.py (hunk 1 above)
Comment (t-wissmann) [reply tests/test_ewmh.py]: Leftover old (and non-working) approach for moving the mouse. :)
I've removed it, thanks!

Comment (blarz) [reply tests/test_ewmh.py]: Thanks!

**Review by blarz — APPROVED** [review]
More tests, more good :)

**Pull request merged**